augur/curate/rename.py

```diff
@@ -40,9 +41,10 @@ def parse_field_map(field_map_arg: List[str]) -> List[Tuple[str,str]]:
         # Sanity check the requests to catch typos etc
         if not old_name:
             raise AugurError(f"The field-map {field!r} doesn't specify a name for the existing field.")
-        if not new_name:
-            raise AugurError(f"The field-map {field!r} doesn't specify a name for the new field.")
-        if new_name in seen_new:
+        if not new_name and not force:
+            raise AugurError(f"The field-map {field!r} doesn't specify a name for the new field."
+                              " If you mean to drop this field you must specify --force.")
```

Comment:
I'm slightly worried that this is conflating the `--field-map`/`--force` flag. 

Thoughts on adding an explicit `--drop-fields` column?

Reply:
To be clear, is the suggestion here to:

1. drop the `--force` flag, and
2. make `--field-map` require both old and new names (and throw an error if new name is missing), and 
3. add an explicit `--drop-fields` option that does what it says on the tin?

If I've got that right, I think it's a sound suggestion — it's more explicit around what it's doing, and more bullet-proof against something like an accidental botched config edit.

Reply:
I'm new to the curate world so I won't push against this. How about we just drop [650bd56](https://github.com/nextstrain/augur/pull/1506/commits/650bd56e907e8fd96ad07c1424d97d1152800ead) which will remove the drop functionality and shift this to a new issue. The resulting situation will require that `--field-map` has non-empty new and old fields (see tests). Note that `--force` is still used for the situation where we overwrite an existing column name (see tests).

Reply:
> How about we just drop [650bd56](https://github.com/nextstrain/augur/pull/1506/commits/650bd56e907e8fd96ad07c1424d97d1152800ead) which will remove the drop functionality and shift this to a new issue.

Sounds good to me. I can write up a separate issue.

Reply:
Commit dropped in force-push

Reply:
Proposed drop column function in https://github.com/nextstrain/augur/issues/1526